libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -127,3 +129,14 @@ void OpticalFlow::update(void)
     _flags.healthy = (AP_HAL::millis() - _last_update_ms < 500);
 }
 
+// singleton instance
+OpticalFlow *OpticalFlow::_singleton;
+
+namespace AP {
+
+OpticalFlow *opticalflow()
+{
+    return OpticalFlow::get_singleton();
```

Comment:
can we normalize on the name? Some times we have `get_instance()`, some times `get_singleton()`, some times `instance()`

Reply:
I have a strong preference for `get_singleton()` or `singleton()`. Instance is very overloaded in any of the drivers that have backends (IE GPS/airspeed/compass)